tools/cell_census_builder/README.md

```diff
@@ -1,16 +1,96 @@
 # README
 
-This is a tool to build the SOMA instantiation of the Cell Census schema, as specified in this doc:
+This package contains code to build and release the Cell Census in the SOMA format, as specified in the
+[data schema](https://github.com/chanzuckerberg/cell-census/blob/main/docs/cell_census_schema.md).
 
-https://docs.google.com/document/d/1GKndzCk9q_1SdYOq3BeCxWgp-o2NSQkEmSBaBPKnNI8/
+This tool is not intended for end-users - it is used by the CELLxGENE team to periodically create and release all
+CELLxGENE data in the above format. The remainder of this document is intended for users of the
+build package.
 
-CAVEATS (READ THIS):
+Please see the top-level [README](../../README.md) for more information on the Cell Census and
+using the Cell Census data.
 
-1. The code is written to the still-rapidly-evolving and **pre-release** Python SOMA API, _and will be subject to change_ as the SOMA API and `tiledbsoma` evolve and stabilize.
-2. The schema implemented by this code is still evolving and subject to change.
-3. The `cell_census_builder` package requires Python 3.9 or later.
+## Overview
 
-## Usage
+This package contains sub-modules, each of which automate elements of the Cell Census build and release process.
+They are wrapped at the package top-level by by a `__main__` which implements the Cell Census build process,
+with standard defaults.
+
+The top-level build can be invoked as follows:
+
+- Create a working directory, e.g., `census-build` or equivalent.
+- If any configuration defaults need to be overridden, create a `config.yaml` in the working directory containing the default overrides.
+- Run the build as `python -m cell_census_builder your-working_dir`
+
+This will perform four steps (more will be added the future):
+
+- host validation
+- build soma
+- validate soma
+- build reports (eg., summary)
+
+This will result in the following file tree:
+
+```
+working_dir:
+    |
+    +-- config.yaml        # build config (user provided, read-only)
```

Comment:
Could be useful to provide an example config.yaml that can be copied as necessary.

Reply:
sure. later info in this doc points the user at the build_state.py file.  And by default you should _not_ specify a config file if doing the standard census build.  It is only for dev reasons that you would use it.

Reply:
rename to `dev-config.yaml`?

Reply:
I would rather leave as-is.  It is the config, for any purpose. I just have defaults set so that it isn't required in our current workflow, but I can easily imagine situations where it is used in the future.